allergies/allergies.py

```diff
@@ -13,30 +13,18 @@
 class Allergies(object):
 
     def __init__(self, score):
-        self.score = score
-        self.allergen_list = self.lst
+        self.allergen_list = []
+        self.score = score % 256
 
-#    @property
+    @property
     def lst(self):
-        allergen_list = []
-        score = self.score % 256
         factors = 7
         while factors >= 0:
-            if score >= 2**factors:
-                score -= 2**factors
-                allergen_list.insert(0, ALLERGENS.get(2**factors))
+            if self.score >= 2**factors:
+                self.score -= 2**factors
+                self.allergen_list.insert(0, ALLERGENS.get(2**factors))
```

Comment:
Though syntactically correct, that isn't performant. The way some list methods are implemented make `insert(0, el)` way slower than using `append` and reversing the whole list at the end with `reverse`. Here is a bit more information on the performance comparison - https://mail.python.org/pipermail/python-list/2004-August/284249.html. 

Also, some syntax isn't really "pythonic". IDK but it's just that I've never seen `insert` in any code base that I've ever read 🤷‍♂️

Reply:
Ah! https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertCountEqual doesn't check for order. So just use `append` here 💥

Reply:
ok, to be honest i have changed it from `append` only to have order right - if it is not needed i go back to `append`